packages/elements-core/src/components/Docs/HttpService/HttpService.tsx

```diff
@@ -15,45 +17,76 @@ import { ServerInfo } from './ServerInfo';
 
 export type HttpServiceProps = DocsComponentProps<Partial<IHttpService>>;
 
-const HttpServiceComponent = React.memo<HttpServiceProps>(({ data, location = {}, layoutOptions, exportProps }) => {
-  const { search, pathname } = location;
-  const mocking = React.useContext(MockingContext);
-  const query = new URLSearchParams(search);
-  return (
-    <Box mb={10}>
-      {data.name && !layoutOptions?.noHeading && (
-        <Flex justifyContent="between" alignItems="center">
-          <Heading size={1} mb={4} fontWeight="semibold">
-            {data.name}
-          </Heading>
-          {exportProps && !layoutOptions?.hideExport && <ExportButton {...exportProps} />}
-        </Flex>
-      )}
-      {data.version && (
-        <Box mb={5}>
-          <VersionBadge value={data.version} />
-        </Box>
-      )}
-      {pathname && layoutOptions?.showPoweredByLink && (
-        <PoweredByLink source={data.name ?? 'no-title'} pathname={pathname} packageType="elements" layout="stacked" />
-      )}
-      <VStack spacing={6}>
-        <ServerInfo servers={data.servers ?? []} mockUrl={mocking.mockUrl} />
-        <Box>
-          {data.securitySchemes?.length && (
```

Comment:
I fixed a little bug here - one should never `&&` off of a numeric value when rendering react stuff. When the value is `0` it ends up rendering the `0`...

<img width="517" alt="Screen Shot 2022-09-14 at 3 27 44 PM" src="https://user-images.githubusercontent.com/847542/190255933-d2f3d735-dd66-4047-800a-ee0ae4e9e667.png">